Field not found: Roles

```text
$ go run ./examples/scanner
panic: failed to enable BLE stack: MapToStruct: Field not found: Roles

goroutine 1 [running]:
main.must(...)
	/dev/shm/bluetooth/examples/scanner/main.go:23
main.main()
	/dev/shm/bluetooth/examples/scanner/main.go:11 +0x1df
```

os: Archlinux
kernel: 5.8.10
bluez: 5.55
go:  1.15.2

--- a/gatts_linux.go
+++ b/gatts_linux.go
@@ -24,6 +24,13 @@ func (a *Adapter) AddService(s *Service) error {
 		return err
 	}
 
+	// disable magic uuid generation because we send through a fully formed UUID.
+	// muka/go-bluetooth does some magic so you can use short UUIDs and it'll auto
+	// expand them to the full 128 bit uuid.
+	// setting these flags disables that behavior.
+	app.Options.UUIDSuffix = ""
+	app.Options.UUID = ""
+
 	bluezService, err := app.NewService(s.UUID.String())
 	if err != nil {
 		return err

